packages/terra-paginator/src/ControlledPaginator.jsx

```diff
@@ -222,7 +222,7 @@ class Paginator extends React.Component {
     );
 
     const navigationMessage = (
-      <VisuallyHiddenText aria-live="polite" aria-relevant="all" text={this.state.selectedPageMessage} />
+      <VisuallyHiddenText aria-live="polite" aria-relevant="text" text={this.state.selectedPageMessage} />
```

Comment:
Does it have any impact on Response of VO in Safari browser..?

Reply:
Tested via VO in Safari. Overall the behavior is expected but looks like nothing would be read out when it comes to the first navigation of the paginator.
Updated the value to "additions text". https://github.com/cerner/terra-core/pull/3927/commits/0d2d11f687340b6071f28dc9d1e2a06d7a72b264 Tested on both VO and JAWS. Everything looks good.